.chloggen/add_paging_device_attr.yaml

```
@@ -0,0 +1,22 @@
+# Use this changelog template to create an entry for release notes.
+#
+# If your change doesn't affect end users you should instead start
+# your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'enhancement'
```

nit:

```suggestion
change_type: 'breaking'
```

Adding a new  metric attribute is a breaking change - https://github.com/open-telemetry/semantic-conventions/issues/722, which is fine on experimental metrics, we just need to document it accordingly